### ISSUE TITLE
ArgumentError: bundle_identifier is missing

### ISSUE THREAD

**Issue by xmarlox** [joemasilotti/railsdevs.com#668]
## Describe the bug

Encountered the error `ArgumentError: bundle_identifier is missing` upon running `bin/setup`

## To reproduce

1. Run `bin/setup` for the first time (in case you already have existing database setup, run first `rails db:drop`)
2. See example logs
```
...
== Preparing database ==
Created database 'railsdevs_development'
Created database 'railsdevs_test'
Seeding development database...
Seeded admins
Seeded developers
Seeded businesses
rails aborted!
ArgumentError: bundle_identifier is missing
... /app/models/concerns/messages/notifications.rb:16:in `send_recipient_notification'
... /app/models/concerns/messages/notifications.rb:5:in `save_and_notify'
... /lib/seeds_helper.rb:45:in `block in create_message!'
... /lib/seeds_helper.rb:44:in `create_message!'
... /db/seeds/conversations.rb:5:in `<top (required)>'
... /db/seeds.rb:4:in `load'
... /db/seeds.rb:4:in `seed'
... /db/seeds.rb:16:in `<top (required)>'
```

## Expected behavior

No error will be encountered while running `bin/setup` specifically in seeding the database

## Actual behavior

Encountered the error `ArgumentError: bundle_identifier is missing` specifically after the seed for businesses

## Additional context

macOS Big Sur
non-Apple M1

**Comment by joemasilotti** [comment]
Thanks for documenting this, @xmarlox! Can you please give #671 a try and let me know if that fixes the issue for you?

**Comment by rayhanw** [comment]
Oh I was having this issue also (on both WSL2 and M1), I got around it by running these 2 manually in `rails c`:
```rb
seed "businesses"
seed "conversations"
```

I tested #671 and it works for me 👍

**Comment by xmarlox** [comment]
@joemasilotti yeah #671 fixed this issue
thanks!

**Comment by joemasilotti** [comment]
Excellent, thanks for the feedback folks. I'll merge that in.

**Comment by joemasilotti** [comment]
Closed via #671.
